app/controllers/api/v1/subjects_controller.rb

```diff
@@ -71,6 +71,12 @@ def destroy
     end
   end
 
+  def surrounding
+    skip_policy_scope
+
+    subject.get_surrounding(params[:subject_set_id], params[:window], params[:gap])
```

Comment:
I think the `get_surrounding` behaviour should be on the `SetMemberSubject` model as that is where the query hits.

Reply:
Also can the method name be `neighbours` ?

Reply:
a) that makes sense, send the subject_set_id and params to this method on the SMS instead

b) why? I asked in the PR desc if you might not rather it was `adjacent`, but I'm not sure `neighbours` is better, even setting aside the US/UK spellings.

Reply:
no great reason - i'd like to ensure the end point naming conveys the direct line from the known subject location in the priority line to it's direct neighbours and i'd like to move away from a verb tense to a noun if we can. 

`surrounding` works, `adjacent` is good but less common. I'm trying to convey as much info and simplify the naming as we can.